utils/get_abstracts.py

```diff
@@ -0,0 +1,84 @@
+import synapseclient
+from synapseclient import Table
+import argparse
+import pandas as pd
+import requests
+import json
+
+
+### Login to Synapse ###
+def login():
+
+    syn = synapseclient.Synapse()
+    syn.login()
+
+    return syn
+
+
+def get_args():
+
+    parser = argparse.ArgumentParser(
+        description='Get synapse publications table id')
+    parser.add_argument('table_id',
+                        type=str,
+                        help='Synapse table id to upload manifest to.')
+
+    return parser.parse_args()
+
+
+def get_df(syn, publications_table_id):
+
+    pubs_query = (f"SELECT pubMedId, abstract FROM {publications_table_id}")
+    pubs_df = syn.tableQuery(pubs_query).asDataFrame().fillna("")
+
+    pmid_list = pubs_df['pubMedId'].tolist()
```

Comment:
This line is considered unnecessary, since it doesn't get used.

Reply:
Ah yes!